tests/Feature/ProductTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+use App\Models\Product;
+
+
+it('user can add a new product', function () {
+
+    userLogin();
+
+    $product = Product::factory()->create();
+
+    $this->post(route('product.store'), [
+        'name' => 'PDU',
+        'brand_id' => $product->brand_id,
+        'size_id' => $product->size_id,
+        'color_id' => $product->color_id,
+        'status' => '0',
+    ])
+        ->assertStatus(302)
+        ->assertRedirect(route('product.index'));
+
+    $this->assertDatabaseHas('products', [
+        'name' => 'PDU',
+        'brand_id' => $product->brand_id,
+        'size_id' => $product->size_id,
+        'color_id' => $product->color_id,
+        'status' => '0',
+    ]);
```

Comment:
why use product factory. just required factory.
make command array to use in pass for route & database checking. because common data type two times.